edit/location.go

```diff
@@ -215,3 +219,11 @@ var _ = RegisterVariable("loc-pinned", func() eval.Variable {
 func (ed *Editor) locPinned() eval.List {
 	return ed.variables["loc-pinned"].Get().(eval.List)
 }
+
```

Comment:
As said above, this should only be a temporary solution. Prefix the variable with a dash and make it `-loc-ignore-case`.